src/main/kotlin/com/manimdsl/linearrepresentation/LinearRepresentation.kt

```diff
@@ -392,34 +373,23 @@ data class TreeNodeRestyle(
     }
 }
 
+/** Miscellaneous **/
+
```

Comment:
If data structures are moved out to their own files. These can be like utility or something more important than miscellaneous